src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -473,13 +473,13 @@ private function validateAndDenormalize(string $currentClass, string $attribute,
                 if (null !== $collectionKeyType = $type->getCollectionKeyTypes()) {
                     [$context['key_type']] = $collectionKeyType;
                 }
-            } elseif ($type->isCollection() && null !== ($collectionValueType = $type->getCollectionValueTypes()) && \count($collectionValueType) > 0 && Type::BUILTIN_TYPE_ARRAY === $collectionValueType[0]->getBuiltinType()) {
+            } elseif ($type->isCollection() && \count($collectionValueType = $type->getCollectionValueTypes()) > 0 && Type::BUILTIN_TYPE_ARRAY === $collectionValueType[0]->getBuiltinType()) {
```

Comment:
the argument of count() can never be null? (same below)

Reply:
This method has been introduced in `symfony/property-info: 5.3` and the method is typehinted to return an array.

https://github.com/symfony/symfony/blob/3581145643f92dba52383c7978cfa3c03ab6c9c1/src/Symfony/Component/PropertyInfo/Type.php#L201